ApolloBuilder.go

```diff
@@ -565,10 +568,8 @@ func (b *Apollo) Complete() (*Apollo, error) {
 	for _, utxo := range b.preselectedUtxos {
 		selectedAmount = selectedAmount.Add(utxo.Output.GetValue())
 	}
-	for _, mintUnit := range b.mint {
-		selectedAmount = selectedAmount.Add(mintUnit.ToValue())
-	}
-
+	burnedValue := b.GetBurns()
```

Comment:
this is the only change that seems suspicious to me; shouldn't it account for both mints and burns?

Reply:
It works in practice. I think it's because any positive mints will be accounted for in the list of payments, so they will be discovered during balancing automatically. This is not the case for txes with net burns

Reply:
Hypothesis: apollo will try to compute the funds needed for a change output to balance the tx, but it simply allows txes where token outputs exceed inputs. That would make the explanation I gave up there make sense. But I'm not 100%

Reply:
I believe there is a more robust implentation of this upstream, I could try pulling that stuff in

Reply:
Since this works for me in practice, and the commit introduces a test case demonstrating the desired behavior, can we merge this and follow up with a PR for the more complete balancing functionality?